CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Excludes public API (declared with public visibility in the interface section).
 - **API**: `VarSectionNode::isThreadVarSection` method.
 - **API**: `ConstSectionNode::isResourceStringSection` method.
+- **API**: `AttributeListNode::getAttributeTypes` API method.
```

Comment:
No need for the redundant "API" in "API method", since we now prefix these changelog entries with **API**.